image/May-2023-image-minutes.md

```diff
@@ -0,0 +1,47 @@
+## 2023-05-31 CAC-DC-Image Meeting
+
+* Date and time: [May 31, 2023, 20:00-20:30 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=CAC+Meeting+DC+Image+Processing&iso=20230531T20&p1=1440&ah=1)
+
+### Attendents
+
+* Sunil Shende (Timekeeper) - SS
+* Josh Moore (Notetaker) - JM
+* Erick Ratamero - ER
+* Toby Hodges - TH
+* Ulf Schiller (Facilitator) - US
+
+### Meeting roles
+
+* Facilitator - Introduces each agenda item. Monitors both the chat and the visual meeting window for “hands”, keeps track of order, and says whose turn it is to speak. Makes sure everyone has a chance to share their views.
+* Notetaker - Records meeting attendance and major points of discussion. Especially takes note of decisions and action items.
+* Timekeeper - Pays attention to the clock and notifies the group when time for specific agenda items is running short.
+
```

Comment:
I'd suggest to remove the generic meeting roles:

```suggestion
```

Reply:
Thanks @uschille!